portal-api/src/v1/api.js

```diff
@@ -268,36 +267,35 @@ const saveUtilisationReport = async (reportData, month, year, user, fileInfo) =>
   }
 };
 
-const getUtilisationReports = async (bankId, month, year) => {
+/**
+ * Gets utilisation reports for a specific bank. If a report
+ * period is not provided, all reports for that bank are
+ * returned. If a report period is provided, the report
+ * submitted for that report period is returned. Returned
+ * reports are ordered by year and month ascending.
+ * @param {string} bankId
+ * @param {import('../types/utilisation-reports').ReportPeriod} [reportPeriod]
+ */
+const getUtilisationReports = async (bankId, reportPeriod) => {
   try {
     if (!isValidBankId(bankId)) {
-      console.error('Get utilisation reports failed with the following bank ID %s', bankId);
-      throw new Error('Invalid bank ID provided: %s', bankId);
+      console.error('Get utilisation reports failed with the following bank ID: %s', bankId);
+      throw new Error(`Invalid bank ID provided: ${bankId}`);
```

Comment:
String substitution please.

Reply:
What's the benefit of using string substitution here? I'm keen to read about this if you have any more info

Reply:
https://console.spec.whatwg.org/#formatter